### PR TITLE
Progress Bar Added 

### PR DESCRIPTION
**Pull request by arhab2001**
I have added Progress bar

### CODE REVIEW

**Review by avincodes — CHANGES_REQUESTED** [review]
Please add a readme file with a screenshot of your snippet in action!

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Screenshot
```
Comment (avincodes) [inline Readme.md:1]: - [ ] Make the title the name of your snippet.

**Review by avincodes — CHANGES_REQUESTED** [review]

**Comment by Safnaj** [comment]
@avincodes better to add these as guidelines in the main readme file rather than forcing the contributors in the pull requests. It will make ur work easy though.

**Comment by avincodes** [comment]
Yep! I have a contributors guidelines file to be created as an issue @Safnaj!

> @avincodes better to add these as guidelines in the main readme file rather than forcing the contributors in the pull requests. It will make ur work easy though.

**Review by avincodes — APPROVED** [review]
Good to go!

**Comment by avincodes** [comment]
Thank you for your pull request @arhab2001!

**Pull request merged**